docs/.vitepress/components/Article.vue

```diff
@@ -1,20 +1,21 @@
 <script setup lang="ts">
 defineProps<{
-	title: string;
+	title?: string;
 	tag?: string;
 	url: string;
 	img: string;
 	author?: string;
 	date?: string;
 	desc?: string;
+	alt?: string;
 }>();
 </script>
 
 <template>
 	<div class="article">
 		<a :href="url">
 			<div class="image">
-				<img :src="img" :alt="title" loading="lazy" />
+				<img :src="img" :alt="alt || ''" loading="lazy" />
```

Comment:
To get ahead of this one - it is not useful to screenreaders to have the title as the alt and then the title immediately after.